ydb/mvp/meta/example-config.yaml

```diff
@@ -13,7 +13,9 @@ generic:
   auth:
     token: your_ydb_token
     token_file: path/to/ydb_token
+    access_service_type: "yandex_v2"
```

Comment:
why do we still need this auth section here? we now have token name in meta database.

maybe there should be a default token name to access cluster's endpoints? but that's it.

Reply:
we need both `token_file` and `access_service_type` for tokenator configuration